src/django/oar/settings.py

```diff
@@ -43,6 +43,11 @@
     raise ImproperlyConfigured(
         'Invalid BATCH_JOB_DEF_NAME provided, must be set')
 
+EXTERNAL_DOMAIN = os.getenv('EXTERNAL_DOMAIN')
+if EXTERNAL_DOMAIN is None:
```

Comment:
I think I steered you a bit wrong in our pairing session. To avoid raising when we run the unit tests on staking we want to skip over this check
```
if EXTERNAL_DOMAIN is None and ENVIRONMENT != 'Development':
```

Reply:
I was just rerunning this test locally, and it did succeed. That seems like a reasonable solution to not overcomplicate things.